### ISSUE TITLE
add ability to create 'RxFormControl' using RxFormBuilder

### ISSUE THREAD

**Issue by AliYusuf95** [rxweb/rxweb#268]
### Description
In some cases, I want to create a FormControl or FormArray. But by using RxFormBuilder I'm able only to create a FormGroup then use the `.get()` to access to the desired object.

```ts
const ctrlObject:  = this.formBuilder
    .group({
        someCrtl: '',
    })
    .get('someCrtl') as FormControl;
```

### Describe the solution you'd like

add `.control()` and `.array()` functionality to RxFormBuilder to be same as `@angular/FormBuilder`

**Comment by ajayojha** [comment]
Thanks for sharing your inputs, will be available both of the function in the coming release.

**Comment by ajayojha** [comment]
@AliYusuf95 This feature is available in 1.9.8 release, please update the package.
